packages/subgraph/schema.graphql

```diff
@@ -1,13 +1,256 @@
-type Token @entity {
+type TenderizeGlobal @entity { # Global Values/ TODO: better name?
+  "Fixed ID of '1'"
   id: ID!
-  decimals: Int
-  name: String
-  symbol: String
+  "List of congifs for all integrations"
+  configs: [Protocol!]!
+  # TODO: Add global totals 
 }
 
-type Transfer @entity {
+type Protocol @entity {
+  "Name of the protocol eg. 'livepeer'"
   id: ID!
+  "Tenderizer Data"
+  tenderizer: Tenderizer @derivedFrom(field: "protocol")
+  "TenderFarm Data"
+  tenderFarm: TenderFarm @derivedFrom(field: "protocol")
+  "Paritcipating user list"
+  users: [UserProtocol!] @derivedFrom(field: "protocol")
+
+  "Address of the ERC20 token for integration"
+  steak: String!
+  "Address of Tenderizer"
+  tenderizerAddress: String! # Move to Tenderizer?
+  "Address of ERC20 Tender Token"
+  tenderToken: String!
+  "Address of Elastic Supply Pool"
+  esp: String!
+  "Address of liquidity pool for pair"
+  bpool: String!
+  "Address of TenderFarm contract"
+  tenderFarmAddress: String! # Move to TenderFarm?
```

Comment:
Would suggest creating a `Config` entity

Reply:
Lol, it was a separate entity before, you suggested to merge it.